modules/vmfcore/include/vmf/compressor_zlib.hpp

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2015 Intel(r) Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http ://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#ifndef VMF_COMPRESSOR_ZLIB_HPP
+#define VMF_COMPRESSOR_ZLIB_HPP
+
+#include "compressor.hpp"
+
+namespace vmf {
+
+/*!
+ * \class ZLibCompressor
+ * \brief Compression algorithm that uses ZLib library.
+ * Currently runs Deflate algorithm with default settings.
+ */
+class VMF_EXPORT CompressorZlib : public Compressor
```

Comment:
let's move this header to the `src` folder, since users shouldn't create/use it directly